utils/hct/hcttest.cmd

```diff
@@ -155,7 +155,10 @@ if "%1"=="-clean" (
   shift /1
 ) else if "%1"=="-custom-bin-set" (
   set CUSTOM_BIN_SET=%~2
- shift /1
+  shift /1
+) else if "%1"=="-file-check-dump" (
+  set ADDITIONAL_OPTS=%ADDITIONAL_OPTS% /p:"FileCheckDumpDir=%~2\HLSL"
+  shift /1
```

Comment:
It would be helpful to document this in the `:showhelp` function. The commit comment seems to suggest the flag takes no argument.

Reply:
Thanks for the suggestion.  And, yes the comments above (which aren't commit comments, but PR comments) should be updated to reflect proper option usage.

Reply:
The funny thing with the PR comment is that the source text had `<target-path>` without the backtick quotes around it, and it was interpreted as some unknown html/xml tag so it disappeared.